profile.md

```diff
@@ -214,13 +214,25 @@ The following properties are defined for usage within a Kubernetes deployment:
   "namespace": "testing"
   ```
 
+- `clusterID`<br>
```

Comment:
I don't think you want that `<br>`

Reply:
Actually I do - I wanted it to insert a LF, just like the other ones have.

Reply:
oh weird. why not just have a new line and tab in and have markdown do the right thing? Perhaps it is all the rest that are also wrong :D

Reply:
See: https://gist.github.com/duglin/d2d0ee2bb6761f1187e0bf2f58a5aa39  the last clusterID/busterID examples. Perhaps I'm doing it wrong.